spec/linter-htmlhint-spec.js

```diff
@@ -32,4 +32,26 @@ describe('The htmlhint provider for Linter', () => {
 
     expect(messages.length).toBe(0);
   });
+
+  it('lints an invalid file (bad.html) with a default configuration when no config file is present', async () => {
+    atom.config.set('linter-htmlhint.disableWhenNoHtmhintConfig', false);
+
+    const bad = path.join(__dirname, 'fixtures', 'bad.html');
```

Comment:
Since this path is being used in multiple places, just move it to the top and replace all 3 uses 😉.

Reply:
Agreed 👍